neo/Network/P2P/Peer.cs

```diff
@@ -37,11 +39,13 @@ private class WsConnected { public WebSocket Socket; public IPEndPoint Remote; p
         protected readonly ConcurrentDictionary<IActorRef, IPEndPoint> ConnectedPeers = new ConcurrentDictionary<IActorRef, IPEndPoint>();
         protected ImmutableHashSet<IPEndPoint> UnconnectedPeers = ImmutableHashSet<IPEndPoint>.Empty;
         protected ImmutableHashSet<IPEndPoint> ConnectingPeers = ImmutableHashSet<IPEndPoint>.Empty;
-
+        protected HashSet<IPAddress> TrustedIpAddresses { get; } = new HashSet<IPAddress>();
+        
         public int ListenerPort { get; private set; }
-        protected abstract int ConnectedMax { get; }
-        protected abstract int UnconnectedMax { get; }
-        protected virtual int ConnectingMax => ConnectedMax - ConnectedPeers.Count;
+        public int MinDesiredConnections { get; private set; } = DefaultMinDesiredConnections;
+        public int MaxConnections { get; private set; } = DefaultMaxConnections;
+        protected int UnconnectedMax { get; } = 1000;
```

Comment:
Good to limit that here as well.

Reply:
This is the same limit as before, it was actually just moved from being defined in LocalNode; there was no advantage to defining it there currently.